Readme.md

```diff
@@ -33,11 +33,29 @@ To check the version:
 
 `tstbtc --help` view the application help
 
-`tstbtc {video_id} {directory}` create video transcript supplying the id of the YouTube video and the source/year
+`pip3 uninstall tstbtc` to uninstall the application
+
+## Usage
+
+`tstbtc {video_id} {directory}` create video transcript supplying the id of the YouTube video and the associated directory bitcointranscripts destination folder
 
 `tstbtc {audio_url} {directory} --title {title}` create audio transcript supplying the url of the audio, the source/year and the title of the audio
 
-`pip3 uninstall tstbtc` to uninstall the application
+To include optional metadata in your transcript, you can add the following parameters:
+- `-t` or `--title`: Supply transcribed file title in 'quotes'
+- `-d` or `--date`: Supply the event date in format 'yyyy-mm-dd'
+- `-T` or `--tags`: Supply the tags for the transcript in 'quotes' and separated by commas
+- `-s` or `--speakers`: Supply the speakers for the transcript in 'quotes' and separated by commas
+- `-c` or `--category`: Supply the category for the transcript in 'quotes' and separated by commas
+
+#### Examples
+To transcribe [this podcast episode](https://www.youtube.com/watch?v=Nq6WxJ0PgJ4) from Stephan Livera's podcast with the associated metadata, we would run either of the below commands. The first uses short argument tags, while the second uses long argument tags. The result is the same.
+- `tstbtc Nq6WxJ0PgJ4 bitcointranscripts/stephan-livera-podcast -t 'OP_Vault - A New Way to HODL?' -d '2023-01-30' -T 'op_vault' -s 'stephan livera, James O’Beirn'  -c ‘podcast’`
+- `tstbtc Nq6WxJ0PgJ4 bitcointranscripts/stephan-livera-podcast --title 'OP_Vault - A New Way to HODL?' --date '2023-01-30' --tags 'op_vault' --speakers 'stephan livera, James O’Beirn'  --category ‘podcast’`
+
+To transcribe [this mp3 link](https://anchor.fm/s/7d083a4/podcast/play/64348045/https%3A%2F%2Fd3ctxlq1ktw2nl.cloudfront.net%2Fstaging%2F2023-1-1%2Ff7fafb12-9441-7d85-d557-e9e5d18ab788.mp3) from Stephan Livera's podcast with the associated metadata, we would run the below command.
+- `tstbtc https://anchor.fm/s/7d083a4/podcast/play/64348045/https%3A%2F%2Fd3ctxlq1ktw2nl.cloudfront.net%2Fstaging%2F2023-1-1%2Ff7fafb12-9441-7d85-d557-e9e5d18ab788.mp3 bitcointranscripts/stephan-livera-podcast --title 'SLP455 Anant Tapadia - Single Sig or Multi Sig?' --date '2023-02-01' --tags 'multisig' --speakers 'stephan livera, Anant Tapadia'  --category 'podcast'`
```

Comment:
Any chance we could find a shorter link for the example?

And the speakers' names should be capitalized on all of the examples.

Reply:
Ig we can  add the name capitalization in the CLI rather than asking the user to capitalize the names.

Reply:
Absolutely. That'd be a nice addition but that's not currently a feature or how the app works.

Reply:
@adamjonas, I'll capitalize speaker names. I'll also see if I can find a shorter link.

If I can't find a shorter mp3 link, what do you think about separating the audio example into two parts. First, I give an example of a real audio link. This way, the user can click it and see a working instance of a mp3 url. Second, I can provide an example with a fake mp3 link and make it explicit that the url is shortened for demonstration purposes. It would look something like the below.
_____________________

You can also transcribe a mp3 link, such as the following from Stephan Livera's podcast: https://anchor.fm/s/7d083a4/podcast/play/64348045/https%3A%2F%2Fd3ctxlq1ktw2nl.cloudfront.net%2Fstaging%2F2023-1-1%2Ff7fafb12-9441-7d85-d557-e9e5d18ab788.mp3

For demonstration purposes, let's shorten the link above to the following: https://anchor.fm/e9e5d.mp3. In this scenario, we would run the below command.
- `tstbtc https://anchor.fm/e9e5d.mp3 bitcointranscripts/stephan-livera-podcast --title 'SLP455 Anant Tapadia - Single Sig or Multi Sig?' --date '2023-02-01' --tags 'multisig' --speakers 'Stephan Livera, Anant Tapadia'  --category 'podcast'`